payload/package.json

```
@@ -9,6 +9,7 @@
 	"devDependencies": {
 		"@minecraft/server": "^1.8.0-beta.1.20.50-preview.24",
 		"@minecraft/server-editor": "^0.1.0-beta.1.20.50-preview.24",
+        "@minecraft/vanilla-data": "^1.20.40",
```

```suggestion
        "@minecraft/vanilla-data": "1.20.40",
```

Don't use caret for vanilla-data, because the minecraft version doesn't follow semver.